packages/composables/src/useLandingSearch.test.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, expect, it, vi } from "vitest";
+import { useLandingSearch } from "./useLandingSearch";
+import { shallowMount } from "@vue/test-utils";
+import * as apiExports from "@shopware-pwa/api-client";
+import LandingPageMock from "./mocks/LandingPage";
+
+const Component = {
```

Comment:
you need to wrap it into `defineComponent` to resolve issue with not seeing component methods:
![image](https://github.com/shopware/frontends/assets/13100280/4184f69f-d699-49e7-af9a-4e652f7b49f6)